homeassistant/components/nmbs/sensor.py

```diff
@@ -92,14 +92,19 @@ def __init__(self, api_client, live_station):
         """Initialize the sensor for getting liveboard data."""
         self._station = live_station
         self._api_client = api_client
-
+        self._unique_id = f"nmbs_live_{self._station}"
         self._attrs = {}
         self._state = None
 
     @property
     def name(self):
         """Return the sensor default name."""
-        return "NMBS Live"
+        return f"NMBS Live {self._station}"
+
+    @property
+    def unique_id(self):
```

Comment:
Did not know about this

Reply:
Yeah I just looked at a bunch of examples and was surprised as well.